docs/evm-sidechain/deploy-a-bridge.md

```diff
@@ -0,0 +1,203 @@
+---
+html: deploy-a-bridge.html
+blurb: Follow these steps to deploy a bridge.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# Deploy a Bridge
+
+Before deploying a bridge, familiarize yourself with the parameters and terms used:
+
+- **Currency:** The currency used by the bridge. Depending on the token, the bridge has different setups. If the currency is XRP, the door account is called a _Native Bridge Door_; if the currency is a token, the door account is called a _Token Bridge Door_.
+- **Locking Chain:** The chain where the token originally exists. If the locking chain is an XRPL chain, the bridge IOU issuer isn't the bridge door account. If the locking chain is an EVM chain, the ERC20 owner isn't the bridge door account.
+- **Issuing Chain:** The chain the token is bridged to. If the issuing chain is an XRPL chain, the bridged IOU issuer is the bridge door account. If the issuing chain is an EVM chain, the bridged ERC20 owner is the bridge door account and can mint/burn tokens.
+- **Owners:** The witnesses addresses. For each witness, you must have its address on both chains.
+- **Threshold:** The amount of attestations needed for each operation.
+- **Signautre Reward:** The reward distributed to witnesses for providing attestations.
+- **Minimum Create Account:** For native XRP bridges, this parameter sets the minimum amount of XRP that needs to be locked in order to execute a create account operation.
+
+
+## Create a Safe Account
+
+Before deploying a bridge, you have to create a Gnosis Safe. The safe is the multisig account that manages all of the bridge's assets.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/s9J0516Ta64" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+1. Connect a wallet to the [Gnosis Safe management platform](https://safe-evm-poa-sidechain.peersyst.tech/).
+
+![Connect a Wallet](../../images/evm-connect-wallet.png "Connect a Wallet")
```

Comment:
Better alt text would make this more useful. For example,

```suggestion
![Screenshot of the "Connect a Wallet" interface in the Gnosis Safe management platform. A "Connect a Wallet" navigation item with a "Connect" button is visible underneath an overlay that asks the user to select from "Available Wallets". The available wallets include MetaMask, Safe Mobile, WalletConnect, and others.](../../images/evm-connect-wallet.png "Connect a Wallet")
```

But also, this seems to actually be several steps:

1. Connect to the EVM Sidechain's instance of Gnosis Safe (at the given link)
2. Click the tiny "Connect a Wallet" text in the top navbar
3. Click the "Connect" button in the popup that appears
4. Select a wallet type from the available options
5. Complete the specific steps for the selected wallet type